facerecognition.py

```diff
@@ -137,7 +135,6 @@ def Game():
                     ) and dot.danger:
                     playing = False
         
-        text = 'Text'
         org = (50,50)
         font = cv2.FONT_HERSHEY_COMPLEX

```

Comment:
Very niiice

Reply:
Thank you for catching that. Looks good.